### ISSUE TITLE
Training on arbitary data

### ISSUE THREAD

**Issue by HelloWorldLTY** [TencentARC/LLaMA-Pro#23]
Thanks for your great work. I wonder if it is possible to directly use alpaca_lora or stanford_alpaca to finetune 8B model in arbitary dataset. Can we access the code? Or we directly use block_expand to create a new model and then train that new model? Does this support huggingface version? Thanks.

**Comment by hills-code** [comment]
Yes, you can directly finetune the 8B model with any datasets. You can access the model in the huggingface (https://huggingface.co/TencentARC/LLaMA-Pro-8B). You can use it just like the normal LLaMA model.

**Comment by HelloWorldLTY** [comment]
Thanks, will try.
